src/VisualStudio/Core/Def/Implementation/ProjectSystem/CPS/IProjectShim.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
+{
+    internal interface IProjectShim : IAnalyzerHost
```

Comment:
Why is this called a ProjectShim?  It seems like there should be a much better name that indicates what this interface is for.  Why does it derive from IAnalyzerHost?